src/scene_battle_rpg2k3.cpp

```diff
@@ -760,13 +748,47 @@ void Scene_Battle_Rpg2k3::ProcessActions() {
 		return;
 	}
 
+	const bool wait = !CheckWait();
+
+	if (wait) {
+		return;
+	}
+
 	if (!battle_actions.empty()) {
 		auto* battler = battle_actions.front();
 		if (!battle_action_pending) {
 			// If we will start a new battle action, first check for state changes
 			// such as death, paralyze, confuse, etc..
 			PrepareBattleAction(battler);
+			auto* action = battler->GetBattleAlgorithm().get();
+
+			if (action->GetSource()->GetType() == Game_Battler::Type_Enemy && combo_repeat == 1) {
+				std::string notification = "";
+
+				if (action->GetType() == Game_BattleAlgorithm::Type::Normal && action->GetRepeat() == 2) {
+					notification = "Double Attack";
+				} else {
+					notification = action->GetStartMessage();
+				}
+				ShowNotification(notification);
+				if (action->GetType() == Game_BattleAlgorithm::Type::Skill) {
+					SetWait(15, 50);
+				} else {
+					if (action->GetType() == Game_BattleAlgorithm::Type::Item
+						|| (action->GetType() == Game_BattleAlgorithm::Type::Normal && action->GetRepeat() == 2)
```

Comment:
You can wrap all this in `if (!notification.empty())` and then only check if skill then (15,50) else (10,40)

Reply:
Changed.